apps/src/lab2/levelEditors/aichatSettings/ModelCardFields.tsx

```diff
@@ -16,7 +16,7 @@ const ModelCardFields: React.FunctionComponent = () => {
   return (
     <div className={moduleStyles['model-card-fields']}>
       {MODEL_CARD_FIELDS_LABELS_ICONS.map(([property, label, _]) => {
-        if (property === 'exampleTopics') {
+        if (property === 'exampleTopics' || property === 'isPublished') {
```

Comment:
Just confirming my understanding. This is not a levelbuilder configurable property because:
- if `modelCardInfo` is editable, it is dependent on whether model card is filled out
- if `modelCardinfo` is read-only, it will always be 'published'.

Reply:
Yeah that's what I was thinking...to your earlier comment about levelbuilder / model card info validation, there's probably some work to figure out what the clearest way is to message the relationship between model card info visibilities and whether the presentation view appears. But I feel like we can do that separately?

Reply:
Sounds good. I can create a jira ticket for this then - thanks!

Reply:
https://codedotorg.atlassian.net/browse/LABS-718
@samantha-code - let us know if this is P0 or not. Thanks!

Reply:
This is not a P0 - with the close relationship with Dan we can make sure the comminicate this for pilot but we would need to be clear about this by July when other T&L members will be picking up this lab to write lessons